src/backend/distributed/worker/worker_merge_protocol.c

```diff
@@ -208,91 +261,7 @@ worker_merge_files_into_table(PG_FUNCTION_ARGS)
 Datum
 worker_merge_files_and_run_query(PG_FUNCTION_ARGS)
```

Comment:
Should we drop the SQL function?

Reply:
I guess we decided not to drop the SQL function because rolling upgrades can happen and in that case if coordinator is running a task-tracker query it might error since the function might not exist in the workers but probably rolling upgrades + task-tracker is very unlikely. (Noted by Marco).

Reply:
Thinking about it more, there is probably no point in leaving a SQL function around without the implementation. We should either drop them or leave the implementation in place. My preference would be the former.